pkg/database/database.go

```diff
@@ -76,11 +81,27 @@ func ConnectDB(
 			}
 		}
 
-		stdlib.Register("sqlite3_ext")
-		s, err := sql.Open("sqlite3_ext", strings.Replace(dbURL.String(), "sqlite://", "file:", 1))
+		sql.Register(SQLiteCustomDriverName, &goSqlite3.SQLiteDriver{
+			ConnectHook: func(conn *goSqlite3.SQLiteConn) error {
+				if err := conn.RegisterFunc("regexp", func(re, s string) bool {
+					c, err := regexp.Compile(re)
+					if err != nil {
+						return false
+					}
+
+					return c.MatchString(s)
+				}, true); err != nil {
+					return err
+				}
+				return nil
```

Comment:
you could return the error from `conn.RegisterFunc` directly

Reply:
done.